src/lib.rs

```diff
@@ -228,6 +239,40 @@ pub fn to_i32(self) -> i32 {
 }
 }
 
+impl SerSignature {
+    /// Get a pointer to the underlying data with the specified capacity.
+    pub fn get_data_mut_ptr(&mut self) -> *mut u8 {
+        self.data.as_mut_ptr()
+    }
+
+    /// Get the capacity of the underlying data buffer.
+    pub fn capacity(&self) -> usize {
+        self.data.len()
+    }
+
+    /// Get the len of the used data.
+    pub fn len(&self) -> usize {
+        self.len
+    }
+
+    /// Set the length of the object.
+    pub(crate) fn set_len(&mut self, len: usize) {
+        self.len = len;
+    }
+
+    /// Convert the serialized signature into the Signature struct.
+    /// (This basically DER deserialize it)
```

Comment:
nit: "deserializes" (and I think the "basically" does not add much)

Reply:
Done